ride-share-app.rb

```diff
@@ -0,0 +1,253 @@
+ride_share_trips = [
+    {
+        'DR001' =>  [
```

Comment:
You could also use symbols as keys

```suggestion
        DR001: [
```